tools/codegen/pkg/sippy/json_types.go

```diff
@@ -101,7 +105,7 @@ func QueriesFor(cloud, architecture, topology, featureGate string) []*SippyQuery
 					ColumnField:   "name",
 					Not:           false,
 					OperatorValue: "contains",
-					Value:         fmt.Sprintf("[FeatureGate:%s]", featureGate),
```

Comment:
This is what allowed us to handle both kube and openshift FeatureGate markers in tests.  Can you refactor to ensure we get OCPFeatureGate and FeatureGate tags?